wordle-web/src/scripts/wordsService.ts

```diff
@@ -1,19 +1,31 @@
 export abstract class WordsService {
   static getRandomWord(): string {
-    return this.#words[Math.floor(Math.random() * this.#words.length)]
+    return this.words[Math.floor(Math.random() * this.words.length)]
   }
 
   static isValidWord(word: string): boolean {
-    return this.#words.includes(word)
+    return this.words.includes(word)
   }
 
-  static validWords(): Array<string> {
+  static validWords(word: string): Array<string> {
     //Todo
-    return new Array<string>()
+    const newList = new Array<string>()
+
+    if (word == '') {
+      return newList
+    }
+
+    this.words.forEach((element) => {
+      if (element.startsWith(word)) {
+        newList.push(element)
+      }
+    })
+
+    return newList
   }
 
   // From: https://github.com/kashapov/react-testing-projects/blob/master/random-word-server/five-letter-words.json
-  static readonly #words: string[] = [
+  private static readonly words: string[] = [
```

Comment:
Could have left static and still gotten same results

Reply:
`static` was never removed. If you're referring to the `#`, I know. I just prefer explicitly saying `private` to utilizing `#` to signify a private variable.